### DISCUSSION TITLE
https://greendolphindance.com/2024/01/12/%E9%95%BF%E6%B2%99%E4%BA%BA%E7%9A%84%E9%95%BF%E6%B2%99%E5%9B%9B%E6%97%A5%E6%B8%B8/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#13] [General]
# https://greendolphindance.com/2024/01/12/%E9%95%BF%E6%B2%99%E4%BA%BA%E7%9A%84%E9%95%BF%E6%B2%99%E5%9B%9B%E6%97%A5%E6%B8%B8/

Back to the rough ground!

https://greendolphindance.com/2024/01/12/%E9%95%BF%E6%B2%99%E4%BA%BA%E7%9A%84%E9%95%BF%E6%B2%99%E5%9B%9B%E6%97%A5%E6%B8%B8/

**Comment by hanlin-ren** [comment]
《热衷于网购》、《难易适中》、《歌单不怎么样》。Dirty咖啡永远滴神！
